README.md

```diff
@@ -1,3 +1,17 @@
+---
+
+This library, ADAL for .NET, will no longer receive new feature improvements. Instead, use the new library
+[MSAL for .NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet).
+
+* If you are starting a new project, you can get started with the
+  [MSAL for .NET docs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki)
+  for details about the scenarios, usage, and relevant concepts.
+* If your application is using the previous ADAL for .NET library, you can follow this
+  [migration guide for .NET apps](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-net-migration) and this [migration guide for brokered iOS Xamarin apps](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-net-migration-ios-broker) to update to MSAL for .NET.
```

Comment:
>iOS [](start = 157, length = 3)

soon...like in a few days, we'll have Android support as well, so maybe include both or just Xamarin

Reply:
Will there be an update to the migration guide for Android?

Reply:
@trwalke did the work for that.